packages/@aws-cdk/aws-lambda/lib/code-signing-config.ts

```diff
@@ -0,0 +1,54 @@
+import { IResource, Resource } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { CfnCodeSigningConfig } from './lambda.generated';
+import { SigningProfile } from '@aws-cdk/aws-signer';
+
+export enum UntrustedArtifactOnDeployment {
+  ENFORCE = 'enforce',
+  WARN = 'warn',
+}
+
+export interface ICodeSigningConfig extends IResource {
+  /**
+   * The ARN of Code Signing Config
+   * @attribute CodeSigningConfigArn
+   */
+  readonly codeSigningConfigArn: string;
+
+  /**
+   * The id of Code Signing Config
+   * @attribute CodeSigningConfigId
+   */
+  readonly codeSigningConfigId: string;
+}
+
+export interface CodeSigningConfigProps {
+  signingProfile: SigningProfile,
```

Comment:
I believe this should be a list

```suggestion
  signingProfiles: ISigningProfile[],
```